libyul/AsmParser.cpp

```diff
@@ -175,7 +175,8 @@ Statement Parser::parseStatement()
 	case Token::Comma:
 	case Token::AssemblyAssign:
 	{
-		std::vector<Identifier> variableNames;
+		Assignment assignment;
+		assignment.location = locationOf(elementary);
```

Comment:
Also, maybe it would be a good idea to add a syntax test for a case where it matters. E.g. a situation where the assignment contains an error and the compiler indicates its location?